sources/packages/web/src/components/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -69,12 +69,9 @@ export default defineComponent({
       {} as StudentApplicationOfferingChangeRequestAPIInDTO,
     );
     const approveApplicationOfferingChangeRequest = ref({} as VForm);
-    const formModel = reactive(
-      {} as StudentApplicationOfferingChangeRequestAPIInDTO,
-    );
     const cancel = () => {
       approveApplicationOfferingChangeRequest.value.reset();
-      formModel.studentConsent = false;
+      approveApplicationOfferingChangeRequestModal.value.studentConsent = false;
```

Comment:
Is the reset above not taking care of it?

Reply:
In my understanding, the reset is not working on the checkbox component. I tried by removing the line 74 but in that case the reset fails.